.pyenv.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+script_dir="$(dirname "${BASH_SOURCE[@]}")"
+current_dir="$(pwd)"
+pathdir="$(python -c "import os.path; print(os.path.abspath(os.path.join('$current_dir', '$script_dir')))")"
+echo "Setting PYTHONPATH=$pathdir"
+export PYTHONPATH="$pathdir"
```

Comment:
This is interesting, what is the use case of this script?

Reply:
Since we don't envision installing this python module system-wide, every script which uses `ssg` needs to know its location via `PYTHONPATH`. Scripts in `build-scripts` are passed it via `cmake`, but scripts in `utils` have no way of knowing `ssg`'s location. My aim was to try and pull out as many of the `sys.path` modifications as I could. This way, the location of `ssg` could be changed at a future date without impacting every single script.

Usage would be: `source .pyenv.sh`; then `PYTHONPATH` gets set to a location with `ssg` and any script depending on it can be run after that. It could also potentially be used as part of `cmake` behind a platform flag if I remove `Setting ` from the echo (then usage could be `.pyenv.sh` python /script.py`).

Also with `cmake`,  and an extension to the script, we could do (in pseudo-cmake):

```
if unix:
 set pyenv ${basedir}/.pyenv.sh
else if windows:
 set pyenv ${basedir}/.pyenv.bat
```
...
```
 ${pyenv} ${python executable} args 
```

and the script could be modified run the specified command with the correct PYTHONPATH set. This would help with Windows support eventually, while still allowing us to support old platforms (read: RHEL6/RHEL7) which don't have CMake 3.1 / `cmake -E env` support.

All of this seems like work for future PRs. Thoughts?

Reply:
I personally would prefer something a little more robust than the use of cmd and shell scripts, and I do see a future possibility to installing `ssg` system-wide to further make content writing more accessible to users.

Reply:
@cipherboy please add the condensed intention of the script to its comment.
@redhatrises you have a good point, we could move towards making this an installable package with library part (the `ssg` in this directory) and maybe executable part (contents of `utils`, `build-scripts`).

Reply:
@matejak Is the new comment sufficient? https://github.com/cipherboy/scap-security-guide/blob/a8986890b3d72f717fb59b61271226ff3fcc0b3c/.pyenv.sh